test/index/validator/shared/schema.test.ts

```diff
@@ -22,32 +22,34 @@
  *  OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
  *  OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
  *  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- **/
+ */
 
-'use strict';
+import avsc from 'avsc';
+import chai from 'chai';
 
-const
-	avsc = require('avsc'),
-	chai = require('chai'),
+import { SchemaValidator } from '../../../../src/index/validator/shared/schema';
 
-	expect = chai.expect,
+const expect = chai.expect;
 
-	schemaValidator = require('../../../../lib/index/validator/shared/schema');
+describe('index/validator/shared/schema.ts', () => {
```

Comment:
OPTIONAL: Wouldn't bother putting .ts extensions - it's just noise when every test has this

Reply:
Sorted